util/syspolicy/policy_keys.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package syspolicy
+
+type Key string
+
+const (
+	// Keys with a string value
+	ControlURL Key = "LoginURL"  // default "https://controlplane.tailscale.com"
+	LogTarget  Key = "LogTarget" // default: ?
```

Comment:
We need to figure out the default for this one.

Also we should probably make the defaults into constants.